tests/selenium/conftest.py

```diff
@@ -129,16 +129,22 @@ def __init__(self, id, mixin, col_id, state=_ANY):
     def get(self, row=0):
         self.mixin._wait_for_table(self.id, self.state)
 
-        return self.mixin.find_elements(
-            '#{} {} tbody tr th.dash-header[data-dash-column="{}"]:not(.phantom-cell)'.format(
-                self.id, self.state, self.col_id
+        return self.mixin.find_element(
+            '#{} {} tbody tr:nth-of-type({}) th.dash-header[data-dash-column="{}"]:not(.phantom-cell)'.format(
+                self.id, self.state, row + 1, self.col_id
             )
-        )[row]
+        )
```

Comment:
Cleaned this up a bit. Thankfully this wasn't used anywhere so no tests need to be modified.